src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/helpers/terraform/get_accounts.py

```diff
@@ -0,0 +1,133 @@
+import boto3
+import json
+import os
+from paginator import paginator
+
+
+sts = boto3.client('sts')
+
+master_acc_id = os.environ["MASTER_ACCOUNT_ID"]
+# master_acc_id = "568100899893"
+if("TARGET_OUS" in os.environ):
+    ou_path = os.environ["TARGET_OUS"]
+# ou_path="/core,/sandbox"
+# root_id="r-okhg"
+
+
+def list_organizational_units_for_parent(parent_ou):
+    organizations = get_boto3_client('organizations', 'arn:aws:sts::' + master_acc_id + ':role/OrganizationAccountAccessRole-readonly', 'getaccountID')
+    # organizations = get_boto3_client('organizations', 'arn:aws:sts::' + master_acc_id + ':role/Admin', 'getaccountID')
+    organizational_units = [
+        ou
+        for org_units in organizations.get_paginator("list_organizational_units_for_parent").paginate(ParentId=parent_ou)
+        for ou in org_units['OrganizationalUnits']
+    ]
+    return organizational_units
+
+def get_accounts():
+    # Return an array of objects like this: [{'AccountId':'xxx','Email':''}]
+    account_details = [] # [{'AccountId':'123','Email':''},{'AccountId':'456','Email':''}]
+
+    print("Master Account ID: " + master_acc_id)
+    # Assume a role into the Org Master role to get account ID's and emails
+    organizations = get_boto3_client('organizations', 'arn:aws:sts::' + master_acc_id + ':role/OrganizationAccountAccessRole-readonly', 'getaccountID')
+    # organizations = get_boto3_client('organizations', 'arn:aws:sts::' + master_acc_id + ':role/Admin', 'getaccountID')
+    for account in paginator(organizations.list_accounts):
+        if account['Status'] == 'ACTIVE':
+            account_details.append({
+                'AccountId': account['Id'],
+                'Email': account['Email']
+            })
+    return account_details
+
+def get_accounts_from_ous():
+    parent_ou_id = None
+    account_list = []
+    organizations = get_boto3_client('organizations', 'arn:aws:sts::' + master_acc_id + ':role/OrganizationAccountAccessRole-readonly', 'getaccountID')
```

Comment:
```suggestion
    organizations = get_boto3_client('organizations', f'arn:aws:sts::{master_acc_id}:role/OrganizationAccountAccessRole-readonly', 'getaccountID')
```
Again, session ID is the same as all the others?